apps/src/sites/code.org/pages/public/yourschool.js

```diff
@@ -7,7 +7,11 @@ $(document).ready(showYourSchool);
 
 function showYourSchool() {
   ReactDOM.render (
-    <YourSchool/>,
-    document.getElementById('your-school')
+    <YourSchool
+      alertHeading={$('#your-school').data("parameters-alert-heading")}
+      alertText={$('#your-school').data("parameters-alert-text")}
+      alertUrl={$('#your-school').data("parameters-alert-url")}
+    />,
+    $('#your-school')[0]
```

Comment:
Why change this to use jquery?

Reply:
I figured if I was going to use jquery for the `.data` lines, it felt more consistent to use it immediately below.  No strong feeling though.

Reply:
My own personal feeling is that `$(selector)[0]` is okay (but still kind of gross) when selector is reasonably complicated, but that not using jquery is preferred when you have an element id. Might mostly be personal preference tho.

Reply:
Similarly torn.. just not sure whether it's better to have slightly gross jquery or to mix jquery and native JS in the same function :)